src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -200,6 +200,12 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 handleExit();
             }
 
+            if (commandResult.isShowFavourites()) {
+                handleFavourite();
+            }
+
+            personListPanel.refresh();
```

Comment:
Just to check, would the favorite status not refresh even without the refresh added here?

Reply:
This refresh should not be needed anymore since this was fixed in #72

Reply:
Right thanks for the reminder! I have made the necessary changes.